.github/workflows/gettext.yml

```
@@ -0,0 +1,24 @@
+name: Gettext Updates
+
+on:
+  push:
+    branches: [master]
```

Maybe we should use the actual main branch here

```suggestion
    branches: [dev]
```